pages/CompressPage.py

```diff
@@ -0,0 +1,386 @@
+        button_3.bind('<Leave>', button_3_leave)
+
+
+        button_image_4 = PhotoImage(
+            file=relative_to_assets("button_4.png"))
+        button_4 = Button(
+            master=self,
+            image=button_image_4,
+            borderwidth=0,
+            highlightthickness=0,
+            command=lambda: controller.show_frame("AboutPage"),
+            relief="flat"
+        )
+        button_4.place(
+            x=0.0,
+            y=337.0,
+            width=259.0,
+            height=30.0
+        )
+
+        button_image_hover_4 = PhotoImage(
+            file=relative_to_assets("button_hover_4.png"))
+
+        def button_4_hover(e):
+            button_4.config(
+                image=button_image_hover_4
+            )
+        def button_4_leave(e):
+            button_4.config(
+                image=button_image_4
+            )
+
+        button_4.bind('<Enter>', button_4_hover)
+        button_4.bind('<Leave>', button_4_leave)
+
+
+        self.image_image_2 = PhotoImage(
+            file=relative_to_assets("image_2.png"))
+        self.image_2 = canvas.create_image(
+            584.0672607421875,
+            284.4524688720703,
+            image=self.image_image_2
+        )
+
+        self.image_image_3 = PhotoImage(
+            file=relative_to_assets("image_3.png"))
+        self.image_3 = canvas.create_image(
+            59.800048828125,
+            352.0,
+            image=self.image_image_3
+        )
+
+        self.image_image_4 = PhotoImage(
+            file=relative_to_assets("image_4.png"))
+        self.image_4 = canvas.create_image(
+            58.800048828125,
+            289.0,
+            image=self.image_image_4
+        )
+
+        self.image_image_5 = PhotoImage(
+            file=relative_to_assets("image_5.png"))
+        self.image_5 = canvas.create_image(
+            59.800048828125,
+            226.0,
+            image=self.image_image_5
+        )
+
+        self.image_image_6 = PhotoImage(
+            file=relative_to_assets("image_6.png"))
+        self.image_6 = canvas.create_image(
+            59.800048828125,
+            163.0,
+            image=self.image_image_6
+        )
+
+        self.image_image_7 = PhotoImage(
+            file=relative_to_assets("image_7.png"))
+        self.image_7 = canvas.create_image(
+            380.499755859375,
+            85.39244079589844,
+            image=self.image_image_7
+        )
+
+        self.image_image_8 = PhotoImage(
+            file=relative_to_assets("image_8.png"))
+        self.image_8 = canvas.create_image(
+            390.800048828125,
+            127.0,
+            image=self.image_image_8
+        )
+
+        self.image_image_9 = PhotoImage(
+            file=relative_to_assets("image_9.png"))
+        self.image_9 = canvas.create_image(
+            583.800048828125,
+            388.0,
+            image=self.image_image_9
+        )
+
+        # self.image_image_10 = PhotoImage(
```

Comment:
Could you please remove the comments if not used anymore?

Reply:
it's still in use for throw error message on the GUI